### ISSUE TITLE
Slideshow not working

### ISSUE THREAD

**Issue by jedikalimero** [ruffle-rs/ruffle#14437]
### Describe the bug

This was an automatic slideshow that showed picture after picture for a while and then started over. There is a pause button that when clicked changes to back, play and forward buttons. The slideshow shows the first picture, then the second and then fails. The buttons seem to work correctly but the slideshow fails just as in automatic mode:

### Expected behavior

The slideshow should cycle through several pictures with a cross fade and then start over but it only works with the two first pictures and then shows the third picture under the first picture and stops. This worked correctly in any browser with the Adobe Flash add on.

Also, the player does not resize dynamically with the browser window and some pictures that appear behind are not resized to the height of the flash movie when they should.

### Content Location

http://www.artofwalls.com/foto/index.html
[foto.zip](https://github.com/ruffle-rs/ruffle/files/13697736/foto.zip)

### Affected platform

Browser's extension

### Operating system

Windows 11

### Browser

Mozilla Firefox 120.0.1

### Additional information

attached zip file with the original .fla file and the pictures and xml file used to address them.

**Comment by Lord-McSweeney** [comment]
I tried playing around with it a little, such as resizing the window and clicking forward/back, and other images do seem to load, they just get partially blocked by the first/second image covering them.

**Comment by jedikalimero** [comment]
> I tried playing around with it a little, such as resizing the window and clicking forward/back, and other images do seem to load, they just get partially blocked by the first/second image covering them.

Yes, but it didn't use to be like that in the old days when Adobe Flash worked.

**Comment by Lord-McSweeney** [comment]
I cannot reproduce this issue with the latest version of Ruffle; @artofwalls can you confirm that the issue has been fixed?

**Comment by artofwalls** [comment]
Now it works perfect!
